### ISSUE TITLE
New-UDPage and wildcard Url

### ISSUE THREAD

**Issue by fmichaleczek** [ironmansoftware/universal-dashboard#141]
Steps to reproduce
------------------
Create 2 pages with 2 Url : one with wildcard and another that include previous path.

```
New-UDPage -Url '/Test/*' -Endpoint {
    New-UDHeading -Content { 'Test' }
}

New-UDPage -Url '/Test/Base/Item' -Endpoint {
    New-UDHeading -Content { 'Item' }
}
```

Expected behavior
-----------------
don't know :)

Actual behavior 
---------------

Page is merge. Each part seems to load async.

But I have recurring message :
`20:23:59 CallbackCmdlet Failed to look up variables.`

**This scenario is supported ?**

It's very interesting to create a menu or a header page for subpages.

```

foreach ( $UDPage in $UDDashboard.Pages.Where{ $_.Url -like "/Test/*" } ) {
     $PageUrl = if ( $UDPage.Url[-1] -eq '*' ) { $UDPage.Url.Substring(0, $UDPage.Url.Length-1) } else { $UDPage.Url }
     New-UDElement -Tag a -Attributes @{ href = $PageUrl } -Content { Split-Path $PageUrl -Leaf  }  
 }
```

Environment data
----------------

- Universal Dashboard 1.6.0-beta3
- Windows 10 1709 
- PowerShell 5.1
- Chrome 65.0.3325.181 (Official Build) (64-bit)

**Comment by fmichaleczek** [comment]
CallbackCmdlet Failed to look up variables
https://github.com/ironmansoftware/universal-dashboard/issues/120

**Comment by adamdriscoll** [comment]
Resolved in 2.2.0
